src/org/opendatakit/briefcase/ui/settings/SettingsPanelForm.java

```diff
@@ -52,6 +52,7 @@
   private JButton storageLocationClearButton;
   private JButton storageLocationChooseButton;
   private JCheckBox pullInParallelField;
+  private JCheckBox resumeLasPullField;
```

Comment:
May I offer you a `t`?

Reply:
I'll take it ;)

No traps were introduced in any commit! :D